From c5006faf05bc4b46767d30f42bcb6157836b7540 Mon Sep 17 00:00:00 2001
From: Marcin Kolny <marcin.kolny@gmail.com>
Date: Sat, 17 Sep 2016 22:49:34 +0200
Subject: Fix warnings

---
 gstreamer/gstreamermm/check.cc   |  2 +-

leio: Trimmed patch down to only ThrowIf that makes compilation fail with newer gcc, not just warn

diff --git a/gstreamer/gstreamermm/check.cc b/gstreamer/gstreamermm/check.cc
index d620435..a0d323d 100644
--- a/gstreamer/gstreamermm/check.cc
+++ b/gstreamer/gstreamermm/check.cc
@@ -54,7 +54,7 @@ RefPtr<Element> Check::_Check::setup_element (const ustring& factory)
 
     element = ElementFactory::create_element (factory, factory);
 
-    ThrowIf (element, "Could not create a '" + factory + "' element");
+    ThrowIf (!!element, "Could not create a '" + factory + "' element");
     CheckRefCount (element->gobj(), factory, 1);
 
     return element;
